django/app/Dockerfile

```
@@ -1,8 +1,24 @@
-FROM python:3.7-alpine 
+# syntax=docker/dockerfile:1.4
+
+FROM --platform=$BUILDPLATFORM python:3.7-alpine AS builder
 EXPOSE 8000
 WORKDIR /app 
 COPY requirements.txt /app
 RUN pip3 install -r requirements.txt --no-cache-dir
 COPY . /app 
 ENTRYPOINT ["python3"] 
 CMD ["manage.py", "runserver", "0.0.0.0:8000"]
+
+FROM builder as dev-envs
+RUN <<EOF
+apk update
+apk add git
+EOF
```

```suggestion
RUN apk add --no-cache git
```
(Avoids having apk index cache in the layer)

---

Not sure 🤔 . That's a dev image and having an index up to date may help user to add new tools, wdyt?

---

Yeah, given the nature of the image it's definitely not a big deal